### DISCUSSION TITLE
Support to render mailable inline images

### DISCUSSION THREAD

**Discussion by Lukasss93** [Xammie/mailbook#98] [Ideas]
I suggest to use the `replaceEmbeddedAttachments` method in this PR:

https://github.com/laravel/framework/pull/48292

**Comment by Xammie** [comment]
Whats the issue you are encountering? Am I correct in assuming you can't preview inline images in mailbook?

**Reply by Xammie** [reply]
I think I've fixed this in [mailbook version 1.8.3](https://github.com/Xammie/mailbook/releases/tag/1.8.3). Could you be so kind to test it and report back if this has fixed your issue?

**Reply by Lukasss93** [reply]
exactly:

Left: mailbook
Right: mailpit

![image](https://github.com/user-attachments/assets/d0f5af18-684a-4b2a-8f87-2290a07596d9)

**Reply by Xammie** [reply]
Awesome to see, enjoy! :)

**Comment by Lukasss93** [comment]
Fixed in mailbook 1.8.3.
Thank you!
